runtime/stdlib/test.go

```diff
@@ -779,9 +886,45 @@ var newMatcherFunction = interpreter.NewUnmeteredHostFunctionValue(
 
 const emulatorBackendTypeName = "EmulatorBackend"
 
-var EmulatorBackendType = func() *sema.CompositeType {
```

Comment:
For some reason, the `EmulatorBackendType` var was exported, however it is not used anywhere outside this package, nor in the `cadence-tools/test` repository. That's why I changed it to not be exported (https://github.com/onflow/cadence/pull/2403/files#diff-4709f76365b48c2caca87b859afcc573afb62cb4e6186b9431d2b540972d402fR111)

Reply:
Yeah, looks like it isn't used anywhere else either: https://github.com/search?q=language%3AGo+EmulatorBackendType&type=code&ref=advsearch